content/en/docs/components/pipelines/overview/multi-user.md

```diff
@@ -1,175 +1,179 @@
 +++
-title = "Multi-user Isolation for Pipelines"
-description = "Getting started with Kubeflow Pipelines multi-user isolation"
+title = "Multi-user Isolation"
+description = "How multi-user isolation works in Kubeflow Pipelines"
 weight = 30
 +++
 
-Multi-user isolation for Kubeflow Pipelines is an integration to [Kubeflow multi-user isolation](/docs/components/multi-tenancy/).
+Multi-user isolation for Kubeflow Pipelines is part of Kubeflow's overall [multi-tenancy](/docs/components/multi-tenancy/) feature.
 
-Refer to [Getting Started with Multi-user isolation](/docs/components/multi-tenancy/getting-started/)
-for the common Kubeflow multi-user operations including the following:
+Refer to [Getting Started with Multi-user isolation](/docs/components/multi-tenancy/getting-started/) for the common Kubeflow multi-user operations 
+like [Managing contributors](/docs/components/multi-tenancy/getting-started/#managing-contributors-through-the-kubeflow-ui).
 
-* [Grant user minimal Kubernetes cluster access](/docs/components/multi-tenancy/getting-started/#pre-requisites-grant-user-minimal-kubernetes-cluster-access)
-* [Managing contributors through the Kubeflow UI](/docs/components/multi-tenancy/getting-started/#managing-contributors-through-the-kubeflow-ui)
-* For Google Cloud: [In-cluster authentication to Google Cloud from Kubeflow](/docs/gke/authentication/#in-cluster-authentication)
+{{% alert title="Warning" color="warning" %}}
+Kubeflow makes no security guarantees about profile isolation.
+It is very likely that malicious attempts to infiltrate other user's profiles will succeed.
```

Comment:
It seems not necessary to include this line, the first line is sufficient to deliver the message of no security guarantee.

Reply:
I agree. We should consider removing or at least rewording.

Reply:
Resolved in https://github.com/kubeflow/website/pull/3021/commits/c3bf6da6cbc8e517174b1d91cdbf9e034464eb6e